linodecli/configuration/auth.py

```diff
@@ -0,0 +1,224 @@
+"""
+Helper functions for configuration related to auth
+"""
+
+import os
+import re
+import sys
+import socket
+import webbrowser
+from http import server
+
+import requests
+
+TOKEN_GENERATION_URL = "https://cloud.linode.com/profile/tokens"
+# This is used for web-based configuration
+OAUTH_CLIENT_ID = "5823b4627e45411d18e9"
+# in the event that we can't load the styled landing page from file, this will
+# do as a landing page
+DEFAULT_LANDING_PAGE = """
+<h2>Success</h2><br/><p>You may return to your terminal to continue..</p>
+<script>
+// this is gross, sorry
+let r = new XMLHttpRequest('http://localhost:{port}');
+r.open('GET', '/token/'+window.location.hash.substr(1));
+r.send();
+</script>
+"""
+
+@staticmethod
```

Comment:
Static method decorator may not be needed as this function is not in a class scope.